WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/TilesKotlinScreen.kt

```diff
@@ -49,6 +62,7 @@ internal fun SampleTilesList(context: Context, sampleTiles: Map<Int, Class<out T
             }
         }
     }
+    FocusOnResume(focusRequester = focusRequester)
```

Comment:
nit: I'd probably assume who ever is creating this knows the timing of focus requests.  But focusRequester is a param with a default value, so it isn't clear here.  Maybe either move FocusOnResume up to MainActivity and make focusRequester a mandatory param.  Or the opposite, remove as a param and create in the body of this method.